src/WindowManager.vala

```diff
@@ -1367,6 +1383,7 @@ namespace Gala
 			// if we have a move action, pack that window to the static ones
 			if (moving != null) {
 				var moving_actor = (WindowActor) moving.get_compositor_private ();
+				kill_window_effects (moving_actor);
```

Comment:
Kill all window effects for the moving window that is the window that is moved to another workspace.